getting-started/templates/systemlink-values.yaml

```diff
@@ -48,6 +48,15 @@ global:
   # <ATTENTION> - Set to false if you do not want to manage secrets as part of the Helm installation.
   ##
   deploySecrets: true
+  ## Customize telemetry data collection.
+  ##
+  telemetry:
+    ## Turn telemetry data collection on or off.
+    ##
+    enabled: false
+    ## Url target of OpenTelemetry exporter. Corresponds to OTEL_EXPORTER_OTLP_ENDPOINT.
```

Comment:
Is OTEL_EXPORTER_OTLP_ENDPOINT going to be meaningful for a user?

Reply:
Probably not, though it is a standard environment variable for the OpenTelemetry stack. So if the user is setting up their own OpenTelemetry stack and are looking to configure that variable for our application, it could be a nice hint.

Reply:
Would there be a useful link to include here?

Reply:
[This page](https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_endpoint) would be the official documentation. I don't know if that's helpful to link to here though.